Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '2.3.4'
+ruby '2.3.1'
```

Comment:
Was this a mistake? I think we should keep the higher patch level (2.3.4) instead of downgrading to 2.3.1, but let me know if you think we should use 2.3.1 instead.

Reply:
Oh. Yes, this was by mistake. I have updated the PR.